### PR TITLE
adjust dashboard resources

### PR DESCRIPTION
**Pull request by moosecanswim**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

This will lower the resources for dashboard

2. What **changes to custom.yaml** are required?

none

3. Have you documented any additional setup steps or configurations options?

none needed

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

I tested this in a devinabox instance using k3d

### CODE REVIEW

File: sense/dashboard/values.yaml [hunk 1]
```diff
@@ -45,11 +45,11 @@ dashboard:
   # CPU and memory limits for the dashboard app
   resources:
     requests:
-      memory: '2Gi'
+      memory: '1Gi'
```
Comment (chrisbsmith) [inline sense/dashboard/values.yaml:48]: This is still more than it really needs.  
```suggestion
      memory: 256Mi'
```

File: sense/dashboard/values.yaml [hunk 2]
```diff
@@ -45,11 +45,11 @@ dashboard:
...
       cpu: '1'
```
Comment (chrisbsmith) [inline sense/dashboard/values.yaml:49]:
```suggestion
      cpu: '200m'
```

File: sense/dashboard/values.yaml [hunk 3]
```diff
@@ -45,11 +45,11 @@ dashboard:
...
     limits:
-      memory: '6Gi'
-      cpu: '2'
+      memory: '2Gi'
+      cpu: '1'
```
Comment (chrisbsmith) [inline sense/dashboard/values.yaml:52]:
```suggestion
      memory: '1Gi'
      cpu: '1'
```

**Review by chrisbsmith — CHANGES_REQUESTED** [review]
The dashboard doesn't really need a ton of resources, so we can set the requests fairly low.

**Comment by moosecanswim** [comment]
@chrisbsmith updated

**Review by chrisbsmith — APPROVED** [review]
lgtm

**Pull request merged**